python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx

```diff
@@ -21,20 +21,22 @@ export default memo(function PromptName({ promptId, name, onUpdate }: Props) {
   };
 
   return (
-    <TextInput
-      value={nameInput}
-      label="Prompt Name"
-      className="ghost"
-      variant="unstyled"
-      placeholder="Name this prompt"
-      onChange={onChange}
-      error={
-        getState().prompts.some(
-          (p) => p.name === nameInput && p._ui.id !== promptId
-        )
-          ? "Name already exists"
-          : null
-      }
-    />
+    <Tooltip label="Prompt Name" position="right">
```

Comment:
Just confirming that we want this tooltip to show up in gradio and local editor as well? Same w/ the one in ModelSelector above